VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddCommentFragment.java

```diff
@@ -107,12 +110,15 @@ public void run() {
         DiscussionTextUtils.setEndorsedState(authorLayoutViewHolder.answerTextView,
                 discussionThread, discussionResponse);
 
+        textViewAddComment.setText(R.string.discussion_add_comment_button_label);
         buttonAddComment.setOnClickListener(new View.OnClickListener() {
             public void onClick(View v) {
                 createComment();
             }
         });
         buttonAddComment.setEnabled(false);
+        buttonAddComment.setContentDescription(getString(R.string.discussion_add_comment_button_description));
+        editTextNewComment.setHint(R.string.discussion_add_comment_hint);
```

Comment:
Could these variable attributes be set through data binding in the layout itself?